mobile_verifier/src/coverage.rs

```diff
@@ -219,138 +177,13 @@ impl CoverageDaemon {
         self.coverage_obj_sink.commit().await?;
         transaction.commit().await?;
 
-        // After writing all of the coverage objects, we set their oracle boosting assignments. This is
-        // done in two steps to improve the testability of the assignments.
-        let unassigned_hexes = UnassignedHex::fetch(&self.pool);
-        let boosting_reports =
-            set_oracle_boosting_assignments(unassigned_hexes, &self.hex_boost_data, &self.pool)
-                .await?;
-        self.oracle_boosting_sink
-            .write_all(boosting_reports)
-            .await?;
-        self.oracle_boosting_sink.commit().await?;
+        // Tell the data set manager to update the assignments.
+        self.new_coverage_object_signal.try_send(())?;
```

Comment:
Making sure I understand unwrapping a `try_send` here.
If for some reason multiple coverage objects are received while the downloader is downloading, or processing, this will crash the coverage server. Then we'll unwind all the processes through the task manager. But the downloader won't die until it's done with whatever it's downloading/processing, so when we come up, we're good?

Reply:
this is incorrect and a remnant of me using `send`, this should just `try_send` and discard any errors.
The reasoning here is that since we've already updated the database, if the channel is full then we will process the new hexes anyway